src/components/initial/Initial.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./style.css";
+import perfilIcon from "../../images/profiles/default.png";
+import { FormSignin } from "../Form/Signin/FormSignIn";
+
+export const Initial = () => (
+	<div className="container">
+		<div className="container-title">
+			<h1 className="title">CondoChat</h1>
+		</div>
+		<div className="container-content flex-row">
+			<div className="container-about flex-column">
+				<h2>About</h2>
+				<p>
+					Simple chat project in React. CondoChat allows a connection
+					and communication between two users in an easy and intuitive
+					way! Create an account and start to chat now!
+				</p>
+				<h2>Contributors</h2>
+				<div className="row">
+					<div className="contibutor">
+						<a href="https://github.com/samueljml">
```

Comment:
don't let any url or number fixed on tsx code, add to a const. EG const DEFAULT_PROFILE_URL